[Feat/button component#112] issue 페이지 버튼 컴포넌트 구현

Park-SooYeon opened the pull request:
## 설명
> PR에 대한 간략한 설명을 작성합니다.
> (필수) 코드를 작성한 이유를 추가한다.

- 컴포넌트 재사용성을 고려하여, 라벨 버튼, 마일스톤 버튼, 초록색 버튼으로 컴포넌트 구현
- 라벨 버튼과 마일스톤 버튼의 경우 isCount 변수를 받아서 true면 라벨, 마일스톤 개수를 보여주고 false면 라벨, 마일스톤 개수를 보여주지 않도록 컴포넌트를 구현하였습니다.

## 체크리스트
> PR 작성자와 확인하는 리뷰어가 확인해야 할 체크리스트를 작성합니다.

- [ ] 라벨 버튼과 마일스톤 버튼의 isCount 변수에 따라 라벨, 마일스톤 개수가 보이는지 확인 부탁드립니다.

## 참고자료
> 해당 PR과 관련된 참고자료가 있을 경우 첨부합니다.

## 기타
> 리뷰어에게 상세한 설명이 필요할 경우 추가 자료를 첨부합니다. (ex. 스크린샷)

Park-SooYeon committed:
Feat: Add issueButtons component (#112)

- issue 페이지에 나타나는 버튼 컴포넌트 생성 및 스타일 추가

Park-SooYeon committed:
Feat: Add button icons (#112)

- issue 페이지에 나타나는 버튼 컴포넌트에 사용할 아이콘 추가

Park-SooYeon committed:
Refactor: Separate button component

- 컴포넌트 재사용성을 생각하여 라벨, 마일스톤 버튼과 연두색 버튼 컴포넌트로 분할
- 버튼에서 공통으로 적용되는 style을 theme으로 설정

Park-SooYeon committed:
Refactor: Separate label, milestone button component

- 재사용성을 위해 label, milestone 버튼 컴포넌트 분리

sbyeol3 commented on client/src/Components/IssueButtons.js:
```diff
@@ -0,0 +1,20 @@
+const IssueButtons = () => {
+  return (
+    <ButtonWrapper>
+      <LabelMilestoneButton isCount />
```
`isCount`는 어떤 prop일까용

sbyeol3 commented on client/src/Components/IssueButtons.js:
```diff
@@ -0,0 +1,20 @@
+  return (
+    <ButtonWrapper>
+      <LabelMilestoneButton isCount />
+      <GreenButton title="New Issue" />
```
재사용성을 높이기 위해 prop으로 넘겨주셨군용

sbyeol3 commented on client/src/Components/IssueButtons.js:
```diff
@@ -0,0 +1,20 @@
+  );
+};
+
+export default IssueButtons;
```
버튼은 1개인데 Buttons 군요

sbyeol3 reviewed (commented):
스크린샷도 같이 첨부해주시면 좋을 것 같아요! 고생하셧슴다

Taeg92 commented on client/src/Components/LabelButton.js:
```diff
@@ -0,0 +1,50 @@
+      </LabelButton>
+    );
+  }
+
```
```js
{isCount ? (
 //isCount가 있을 때 component
) : (
 //isCount가 없을 때 component
)}
```
이런식으로 깔끔하게 수정하면 어떨까요?

Park-SooYeon committed:
Retactor: Edit props name

- isCount 변수의 이름이 모호하여 hasCount로 변경
- 라벨, 마일스톤의 개수를 보여주고 싶을 때 사용할 변수로 hasCount 사용

Park-SooYeon committed:
Refactor: Edit code style

- 중복 코드를 줄이기 위하여 코드 스타일 수정
